.github/workflows/build_and_test_workflow.yml

```diff
@@ -25,15 +25,15 @@ env:
   TEST_OPENSEARCH_TRANSPORT_PORT: 9403
   TEST_OPENSEARCH_PORT: 9400
   OSD_SNAPSHOT_SKIP_VERIFY_CHECKSUM: true
-  NODE_OPTIONS: "--max-old-space-size=6144 --dns-result-order=ipv4first"
+  NODE_OPTIONS: '--max-old-space-size=6144 --dns-result-order=ipv4first'
 
 jobs:
   build-lint-test:
     name: Build and Verify on ${{ matrix.name }} (ciGroup${{ matrix.group }})
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest]
```

Comment:
Do we want to do the tests that are done in windows or not?  

I understand from this that several windows tests are not going to be run.